pkg/util/util.go

```diff
@@ -30,6 +30,70 @@ const (
 	osdServiceAccountName = "osd-managed-admin"
 )
 
+// ConfigMapOperations store informations required for operations
+type ConfigMapOperations struct {
```

Comment:
if this is its own type with methods, I'd prefer to move it into its own file, potentially outside of the util folder.

Reply:
+1

Reply:
migrated to `configmap` folder. Can be renamed again, not sure that suits to here well